import_statement.php

```diff
@@ -271,7 +293,8 @@ protected function process_import_row($table, $row, $dupMode, $dupCheckColumns,
             return $GLOBALS['locImportStatementFieldMissing'];
         }
 
-        $refnr = str_replace(' ', '', $row['refnr']);
+        $refnr = str_replace("'", '', $row['refnr']);
+        $refnr = str_replace(' ', '', $refnr);
```

Comment:
Please give str_replace an array of strings to replace as the first parameter to avoid the second call.